### PR TITLE
Aptitude Engine beta

### PR DESCRIPTION
**Pull request by AByteCoder**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+@app.after_request
+def add_header(response):
+    """
+    Add headers to both force latest IE rendering engine or Chrome Frame,
+    and also to cache the rendered page for 10 minutes.
+    """
+    response.headers['X-UA-Compatible'] = 'IE=Edge,chrome=1'
+    response.headers['Cache-Control'] = 'public, max-age=0'
+    return response
+@app.route('/')
```
Comment (theBuzzyCoder) [inline main.py]: Add two newline above 
```python
@app.route
```
Everywhere, wherever you have not added newlines after end of function

File: modules/AptitudeEngine.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+dataset_labels = ["number_series","relative_speed","clock","lcm","TimeWork"]
+dataset_location ="/home/ashish/AptitudeEngine/AptitudeEngine/modules/Dataset/jsons"
+dataset_files = ["numseries.json","train.json","clock.json","lcm.json","TimeWork.json"]
+extract_parameters = {}
+extract_parameters['number_series'] = extract_numbers.NumbersExtractParameters();
+extract_parameters['relative_speed'] = extract_train.TrainsExtractParameters();
+extract_parameters['clock'] = extract_clock.ClockExtractParameters();
+extract_parameters['lcm'] = extract_lcm.LCMExtractParameters();
+extract_parameters['TimeWork'] = extract_work.TimeWorkExtractParameters();
+def format_solution(sol_dict):
```
Comment (theBuzzyCoder) [inline modules/AptitudeEngine.py]: new line

**Review by theBuzzyCoder — CHANGES_REQUESTED** [review]

**Comment by AByteCoder** [comment]
Added newlines in my recent commit

**Review by theBuzzyCoder — APPROVED** [review]

**Comment by theBuzzyCoder** [comment]
This has been merged to the master branch

**Pull request closed without merging**